src/pages/puzzles/[instanceId]/index.tsx

```diff
@@ -38,35 +41,55 @@ const PuzzlePage = ({
     user
   ) => {
     if (isAuthenticated) {
-      const submission = await submitPuzzleInstance(
-        answer,
-        puzzleInstance,
-        puzzle,
-        randomSeed,
-        user
-      );
+      const submissionInstance = async () => {
+        const submission = await submitPuzzleInstance(
+          answer,
+          puzzleInstance,
+          puzzle,
+          randomSeed,
+          user
+        );
 
-      if ((submission as HandledError).error) {
-        // TODO: handle errors from submitting
-        alert(submission.message);
-        return;
-      }
+        if ((submission as HandledError).error) {
+          console.error((submission as HandledError).message);
```

Comment:
console?

Reply:
i think instead of sending the error through toaster for the end-user, I think it's better to send to console (`console.error` will send a message with error format so it's easy to identify)

Reply:
But how would the user see the `console.error()`, wouldn't they have to already have developer tools open to see it?